pilot/pkg/bootstrap/server.go

```diff
@@ -980,9 +980,8 @@ func (s *Server) initDiscoveryService(args *PilotArgs) error {
 				log.Warna(err)
 			}
 		}()
-		if len(args.DiscoveryOptions.SecureGrpcAddr) > 0 {
```

Comment:
what about this check?

Reply:
The length must be >0, because of L964~L967